src/Common/Admin/Settings_Sidebar.php

```diff
@@ -42,64 +37,94 @@ class Settings_Sidebar extends Section {
 	public function render() {
 		?>
 		<div class="tec-settings-form__sidebar">
-		<?php do_action( 'tec_settings_sidebar_start' ); ?>
-			<div class="tec-settings-form__sidebar-section tec-settings-form__sidebar-header">
-				<?php do_action( 'tec_settings_sidebar_header_start' ); ?>
-				<?php
-				$this->render_header_image();
-				$this->render_title();
-				?>
-				<?php do_action( 'tec_settings_sidebar_header_end' ); ?>
-			</div>
-
-			<?php foreach ( $this->sections as $section ) : ?>
+			<?php do_action( 'tec_settings_sidebar_start', $this ); ?>
+			<?php foreach ( $this->get_sections() as $section ) : ?>
 				<div class="tec-settings-form__sidebar-section">
 					<?php $section->render(); ?>
 				</div>
 			<?php endforeach; ?>
-			<?php do_action( 'tec_settings_sidebar_end' ); ?>
+			<?php do_action( 'tec_settings_sidebar_end', $this ); ?>
 		</div>
 		<?php
 	}
 
 	/**
-	 * Set the header image for the sidebar.
+	 * Alias to prepending a section to the sidebar.
 	 *
 	 * @since 6.1.0
 	 *
-	 * @param Image $image The image to set.
+	 * @param Section $section The section to add.
+	 * @param bool    $append  Whether to append the section or prepend it.
 	 *
-	 * @return void
+	 * @return self
 	 */
-	public function set_header_image( Image $image ) {
-		$this->header_image = $image;
+	public function add_section( Section $section, bool $append = true ): self {
```

Comment:
I don't think we should add duality to how this method functions, and I don't think that we need to have both `append` and `prepend` methods. While I understand the use case of prepending a section, we don't have to have symmetry in the methods available. 

With that said, I think we should remove the new `append_section()` method, keep the new `prepend_section()` method, and remove the second parameter to this method.

Reply:
Done. 🫀